com.unity.multiplayer.mlapi/Tests/Runtime/TickSystemTests.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+
+namespace MLAPI.RuntimeTests
+{
+    public class TickSystemTests: IDisposable
+    {
+        private NetworkTickSystem m_TickSystem = null;
+        private float m_TestDuration = 5.0f;
+        private float m_SleepInterval = 0.001f;
+        private float m_TickInterval = 0.010f;
+
+        public void Dispose()
+        {
+            m_TickSystem.Dispose();
+            m_TickSystem = null;
+            NetworkUpdateLoop.UnregisterLoopSystems();
+        }
+
+        [UnityTest]
+        public IEnumerator VerifyTickSystem()
+        {
+            m_TickSystem = new NetworkTickSystem(m_TickInterval);
+
+            ushort tick0 = m_TickSystem.GetTick();
```

Comment:
Standards Suggestion _( minor nit-pick related )_
When you have the time, it would be more standards compliant to replace all of the locally scoped variable declarations as "var" and not the actual type.
Example:
From this:  ushort tick0 = m_TickSystem.GetTick();
To this: var tick0 = m_TickSystem.GetTick();

There are several places this could be applied within VerifyTickSystem